integration_tests/src/main/python/rap_conf.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+confMap = {
+  'spark.master':                               'local',
+  'spark.ui.showConsoleProgress':               'false',
+  'spark.sql.session.timeZone':                 'UTC',
+  'spark.sql.shuffle.partitions':               '12',
+  'spark.executor.extraJavaOptions':            '-Duser.timezone=GMT',
+}
+
+def update_conf_map():
```

Comment:
This is not really what I wanted or asked for. It speeds up the build so much that part of me just wants to merge this in as is and then I can do a follow on PR to get it exactly where I want it.  If you are OK with that I will hit the merge button on Monday.  If you would rather pull it across the finish line yourself I am fine with that, but we need to move all of the configs to environment variables like I asked before

https://github.com/NVIDIA/spark-rapids/pull/1022#discussion_r513411342

and we need to update the documentation 

https://github.com/NVIDIA/spark-rapids/blob/branch-0.3/integration_tests/README.md

to explain about these changes.

Reply:
@revans2 I'm ok that we merge this first to speedup the pre-build pipeline.

As for moving all the configs to environment variables, we can do it in the next PR, and I'm glad if it will not take up your time long.  I was blocked by setup Blossom Jenkins for spark-rapids IT on EMR and dataproc cluster, so sorry for the delayed response to the issue.